client/marketing/components/recommended-extensions/test/index.js

```diff
@@ -47,36 +46,49 @@ describe( 'Recommendations and not loading', () => {
 	} );
 
 	it( 'should not display the spinner', () => {
-		const spinner = recommendedExtensionsWrapper.find( Spinner );
-		expect( spinner.length ).toBe( 0 );
+		const { container } = recommendedExtensionsWrapper;
+		expect(
+			container.getElementsByClassName( 'components-spinner' )
+		).toHaveLength( 0 );
```

Comment:
What do you think of:

```js
expect(
	container.querySelector( '.components-spinner' )
).toBeNull();
```

[`querySelector()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) returns `null` when the selector can't be found.

Reply:
It will work, but iirc the output is more helpful when `not.toBeInTheDocument()` is used. I'd suggest breaking the test and compare the output. It's fine to go with any option you like more :)